zuul.d/nodeset.yaml

```diff
@@ -41,6 +41,18 @@
       - name: peers
         nodes: []
 
+- nodeset:
+    name: centos-stream-9-vexxhost
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+    groups:
+      - name: switch
+        nodes:
+          - controller
+      - name: peers
+        nodes: []
+
```

Comment:
so the content provider will always run in vexx
but what if child jobs get scheduled on ibm cloud don't we have the same problem?

or can ibm child jobs reach the vexx content provider?

Reply:
vexx IPs are public

Reply:
yeah indeed... i guess I was wondering if there was administrative issue with reaching vexx from ibm

Reply:
There shouldn't be an issue reaching Vexxhost from IBM cloud, not sure how to find a job with that scenario but as Vexxhost is a public IP it works.